backend/infrahub/core/constants/graphql_payload.py

```diff
@@ -0,0 +1 @@
+KIND_GRAPHQL_FIELD_NAME = "__kind__"
```

Comment:
let's move it to infrahub/graphql/constants.py
we need to start breaking down the constants into each section of the code